docs/tutorials/plotting/pairplot_with_gmm.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pairplot with GMM: Visualizing High Dimensional Data\n",
+    "\n",
+    "This example provides how to visualize high dimensional data using the pairplot_with_gmm. "
```

Comment:
"using the pairplot_with_gmm" sentence structure is kinda weird

Reply:
any tips on how to make it sound better? @loftusa

Reply:
"using the pairplot_with_gmm function" maybe

Reply:
fix https://github.com/microsoft/graspologic/pull/541/commits/ed819eead52eb3bf208308105c2ad5924a31e2f0